src/proj.h

```diff
@@ -0,0 +1,205 @@
+/******************************************************************************
```

Comment:
Your google-fu is better than mine :-)

I have discussed the subject with @kbevers, and I think the common ground is that we could change the name if  anyone comes up with a good one. proj4.h was mentioned...